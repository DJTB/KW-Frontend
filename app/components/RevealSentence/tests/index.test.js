import React from 'react';
import { shallow } from 'enzyme';
import MaskedSentence from '../index';

describe('<MaskedSentence />', () => {
  it('should match baseline snapshot', () => {
    expect(shallow(<MaskedSentence />)).toMatchSnapshot();
  });
});
