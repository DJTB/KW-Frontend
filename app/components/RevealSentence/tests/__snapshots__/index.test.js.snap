// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MaskedSentence /> should match baseline snapshot 1`] = `
<styled.div
  tabIndex="0"
>
  <styled.p />
  <Styled(Icon)
    name="EYE"
  />
</styled.div>
`;
